setup.sh

```diff
@@ -33,8 +32,8 @@ git init
 git remote add origin https://github.com/hashbang/shell-etc.git
 git fetch --all
```

Comment:
Would prefer `git fetch origin master` here: no need to pull in all the branches on the server.